js/globals.ts

```diff
@@ -61,6 +62,7 @@ export type Blob = blob.DenoBlob;
 // window.File = file.DenoFile;
 // export type File = file.DenoFile;
 
+window.Event = event.Event;
```

Comment:
We should have an `export type Event = event.Event;` to ensure this type exists in the global context for users of Deno.

Reply:
added, all set!